Renaming prior to the delivery of the work about "NFC Commissioning". (#36801)
project-chip/connectedhomeip: docs/platforms/nrf/nrfconnect_examples_configuration.md

-   `CONFIG_CHIP_ICD_SLOW_POLL_INTERVAL`
-   `CONFIG_CHIP_ICD_FAST_POLLING_INTERVAL`

> #### Onboarding with NFC support

Reviewer: How about:
```suggestion
#### Onboarding payload delivered over NFC
```

> You can configure the Matter protocol to use an NFC tag for onboarding, instead
> of using a QR code, which is the default configuration.

Reviewer: ```suggestion
You can configure the Matter protocol to use an NFC tag for delivering the onboarding payload, instead
of using a QR code, which is the default configuration.
```

> To enable NFC for onboarding and share the onboarding payload in an NFC tag, set
> the `CONFIG_CHIP_NFC_ONBOARDING_PAYLOAD` option.

Reviewer: ```suggestion
To enable sharing the onboarding payload in an NFC tag, set
the `CONFIG_CHIP_NFC_ONBOARDING_PAYLOAD` option.
```
